contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,16 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    pack:
```

Comment:
And this: https://github.com/StackStorm/st2/blob/8e32faf5cfbb7ea747f088ef2d82c980e7d978ec/contrib/packs/actions/pack_mgmt/download.py#L170

So we don't, of course. If it's just a name of the pack, like `sensu`, it'll be downloaded from exchange. If it's a git url, and not a name, it'll be downloaded from git.